client/src/App.js

```diff
@@ -1,14 +1,25 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react";
+import logo from "./logo.svg";
+import "./App.css";
+import axios from "axios";
 
 class App extends Component {
+  state = {
```

Comment:
This is purely to demonstrate end-to-end functionality. The next revision will encapsulate all HTTP requests within Redux action creators.